### PR TITLE
Revise conflict data resolution enhancement during data import

### PR DESCRIPTION
**Pull request by lyzx2001**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16861
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ strategy = ""
 # 当后端是 “importer” 时，tikv-importer 的监听地址（需改为实际地址）。
 addr = "172.16.31.10:8287"
 
-# 从 v8.0.0 开始，`duplicate-resolution` 参数已废弃。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
+# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在 v8.5.0 中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:145]:
```suggestion
# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```

File: tidb-lightning/tidb-lightning-error-resolution.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ summary: 介绍了如何解决导入数据过程中的类型转换和冲突错
 
 - `lightning.max-error`：类型错误的容忍阈值
 - `conflict.strategy`、`conflict.threshold`、`conflict.max-record-rows`：数据冲突错误的相关配置
-- `tikv-importer.duplicate-resolution`（从 v8.0.0 开始已被废弃）：物理导入模式下的冲突处理配置
+- `tikv-importer.duplicate-resolution`（从 v8.0.0 开始已被废弃，并将在 v8.5.0 中被移除）：物理导入模式下的冲突处理配置
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-error-resolution.md:18]:
```suggestion
- `tikv-importer.duplicate-resolution`（从 v8.0.0 开始已被废弃，并将在未来版本中被移除）：物理导入模式下的冲突处理配置
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ strategy = ""
 backend = "local"
 
 # 冲突数据处理方式
-# 从 v8.0.0 开始，`duplicate-resolution` 参数已废弃。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
+# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在 v8.5.0 中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:51]:
```suggestion
# `duplicate-resolution` 参数从 v8.0.0 开始已被废弃，并将在未来版本中被移除。详情参考 <https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-physical-import-mode-usage#旧版冲突检测从-v800-开始已被废弃>。
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 4]
```diff
@@ -99,7 +99,7 @@ Lightning 的完整配置文件可参考[完整配置及命令行参数](/tidb-l
 
 冲突数据是指两条或两条以上记录中存在主键或唯一键列数据重复。当数据源中的记录存在冲突数据，如果没有启用冲突数据检测功能，将导致该表的实际总行数与使用唯一索引查询的总行数不一致。
 
-冲突数据检测采用新版冲突检测 (`conflict`) 模式。旧版冲突检测 (`tikv-importer.duplicate-resolution`) 模式从 v8.0.0 开始已被废弃。
+冲突数据检测采用新版冲突检测 (`conflict`) 模式。旧版冲突检测 (`tikv-importer.duplicate-resolution`) 模式从 v8.0.0 开始已被废弃。`tikv-importer.duplicate-resolution` 参数将在 v8.5.0 中被移除。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:102]:
```suggestion
冲突数据检测采用新版冲突检测 (`conflict`) 模式。旧版冲突检测 (`tikv-importer.duplicate-resolution`) 模式从 v8.0.0 开始已被废弃。`tikv-importer.duplicate-resolution` 参数将在未来版本中被移除。
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 5]
```diff
@@ -126,7 +126,7 @@ Lightning 的完整配置文件可参考[完整配置及命令行参数](/tidb-l
 
 ### 旧版冲突检测（从 v8.0.0 开始已被废弃）
 
-从 v8.0.0 起，旧版冲突检测 (`tikv-importer.duplicate-resolution`) 已被废弃。如果 `tikv-importer.duplicate-resolution` 为 `remove` 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 `"replace"` 开启新版冲突检测。需要注意 `tikv-importer.duplicate-resolution` 不能与 `conflict.strategy` 同时配置，否则将报错。
+从 v8.0.0 起，旧版冲突检测 (`tikv-importer.duplicate-resolution`) 已被废弃。`tikv-importer.duplicate-resolution` 参数将在 v8.5.0 中被移除。如果 `tikv-importer.duplicate-resolution` 为 `remove` 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 `"replace"` 开启新版冲突检测。需要注意 `tikv-importer.duplicate-resolution` 不能与 `conflict.strategy` 同时配置，否则将报错。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:129]:
```suggestion
从 v8.0.0 起，旧版冲突检测 (`tikv-importer.duplicate-resolution`) 已被废弃。`tikv-importer.duplicate-resolution` 参数将在未来版本中被移除。如果 `tikv-importer.duplicate-resolution` 为 `remove` 且 `conflict.strategy` 未设置，TiDB Lightning 会自动将 `conflict.strategy` 赋值为 `"replace"` 开启新版冲突检测。需要注意 `tikv-importer.duplicate-resolution` 不能与 `conflict.strategy` 同时配置，否则将报错。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**